website/project/views/node.py

```diff
@@ -49,7 +50,8 @@
 def edit_node(auth, node, **kwargs):
     post_data = request.json
     edited_field = post_data.get('name')
-    value = strip_html(post_data.get('value', ''))
+    value = post_data.get('value', '')
```

Comment:
Why was sanitization removed here? There doesn't seem to be an alternate sanitization mechanism in `node.set_title` or `node.set_description`.